src/v/pandaproxy/schema_registry/service.cc

```diff
@@ -194,6 +215,31 @@ class wrap {
 
     void audit_authz(const server::request_t& rq) const { do_audit_authz(rq); }
 
+    void audit_authz_failure(
+      const server::request_t& rq,
+      const request_auth_result auth_result,
+      ss::sstring reason) const {
+        vlog(
+          plog.trace, "Attempting to audit authz for {}", rq.req->format_url());
+        auto success = rq.service().audit_mgr().enqueue_api_activity_event(
+          security::audit::event_type::schema_registry,
+          *rq.req,
+          auth_result,
+          audit_svc_name,
+          false,
+          std::move(reason));
+
+        if (!success) {
+            vlog(
+              plog.error,
+              "Failed to audit authorization request for endpoint: {}",
+              rq.req->format_url());
```

Comment:
I don't know if this is a new bug, but I'm seeing the query params repeated.

Same for authn (and it has a typo: "Failed to audit authnetication request for endpoint: {}"

Reply:
I fixed the typo, thanks.

The bug existed before as well. I had a quick look but I couldn't find where it's coming from just yet.

Reply:
I think `format_url` :'(